content/agent/faq/downgrade-datadog-agent.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Downgrade the Agent to a prior minor version
+kind: faq
+further_reading:
+---
+
+For DEB or RPM package of the Datadog Agent, find below instructions to downgrade the Datadog Agent to a version `6.<VERSION>`.
```

Comment:
this notation is a bit weird (it's not obvious what `VERSION` should be replaced with), I'd advise using `6.Y.Z` instead for example in all of the example below.

Or, alternatively, keep writing `<VERSION>`, but mention in the intro here that version should be of the form `6.Y.Z`, for example `6.5.1`.